ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -0,0 +1,36 @@
+# global
+import ivy
+
+
```

Comment:
If you could add the following line at top of document

`ToDo: Add this function to functional API`

Reason being if this is a function that exists on all frameworks, we should at some stage add it to all the functional api